src/OpenTelemetry/Logs/LoggerProviderSdk.cs

```diff
@@ -69,7 +69,7 @@ public LoggerProviderSdk(
                 this.instrumentations.Add(instrumentation.Instance);
             }
 
-            instrumentationFactoriesAdded.Append(instrumentation.Name);
+            instrumentationFactoriesAdded.Append($"{instrumentation.Name} {instrumentation.Version}");
```

Comment:
I am not against logging of the Version here. It is important to mention that it will be taking the version in this way `typeof(TInstrumentation).Assembly.GetName().Version?.ToString() ?? "1.0.0.0"`. It means that it will be reporting only major version of the library in most cases (if the library follows [this recomendation](https://learn.microsoft.com/en-us/dotnet/standard/library-guidance/versioning#assembly-version)).

This version will be different than typical data reported by the contrib instrumentation packages https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/89f0af731441a364779693a1874c57f33b1e721c/src/Shared/AssemblyVersionExtensions.cs

Reply:
want me to change it to not record if no version is found?

Reply:
I am telling that it usually will record 1.0.0.0/2.0.0.0/3.0.0.0,etc. There will be no differences between e.g. 1.1.0 nuget package and 1.2.0.

Reply:
then shouldnt it default to use the AssemblyInformationalVersionAttribute it is exists?

Reply:
For sure, it is one of the options, you can consider.